pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter.go

```diff
@@ -329,7 +329,8 @@ func (admitter *VMsAdmitter) validateVolumeRequests(ar *v1beta1.AdmissionRequest
 	}
 
 	newSpec := vm.Spec.Template.Spec.DeepCopy()
-	for _, volumeRequest := range vm.Status.VolumeRequests {
+	for _, volumeRequest_ := range vm.Status.VolumeRequests {
```

Comment:
Nit: I think that shadowing here would make sense. It is also idiomic and recommended in [effective_go ](https://golang.org/doc/effective_go.html#channels) (see shadowing of request)

Reply:
@xpivarc I'm not sure I understand the comment. The problem is the storing of the loop variable address (e.g., 
   curVMAddRequestsMap[name] = &volumeRequest).  In case there are multiple requests this will cause to store the same address for different requests basically pointing to the same requests

Reply:
Oh sorry got you. You mean shadowing instead of creating new "_" variable

Reply:
To be honest, although shadowing is idiomic in go, I get the felling people will not really like it :-) 
I can easily replace everything with shadowing.   WDYT?

Reply:
FWIW I hate this shadowing ever so slightly less than the `variable_` annotation.

Reply:
OK shadowing it is ...

Reply:
I removed most of the variables and switched to use the actual array . Seem more clear